File: app/src/main/java/com/shmedo/mcloudapp/deviceconfig/model/params/QueryCmdResultParam.java
package com.shmedo.mcloudapp.deviceconfig.model.params;

import java.util.List;

/**
 * 创建者:   gonghe <br/>
 * 创建时间:  2021/12/23 <br/>
 * 描述：     查询指令执行结果参数
 */
public class QueryCmdResultParam {
    private List<String> msgIDList;//指令下发后返回的MsgID列表

    public List<String> getMsgIDList() {
        return msgIDList;
    }

    public void setMsgIDList(List<String> msgIDList) {
        this.msgIDList = msgIDList;
    }
}

File: app/src/main/java/com/shmedo/mcloudapp/model/QueryCmdResultParam.kt
package com.shmedo.mcloudapp.model

import com.squareup.moshi.JsonClass

/**
 * 创建者:   gonghe <br></br>
 * 创建时间:  2021/12/23 <br></br>
 * 描述：     查询指令执行结果参数
 */
@JsonClass(generateAdapter = true)
class QueryCmdResultParam(
    val msgIDList: List<String> = arrayListOf() //指令下发后返回的MsgID列表
)
